app/src/main/java/io/homeassistant/companion/android/controls/LightControl.kt

```diff
@@ -40,22 +42,31 @@ class LightControl {
             control.setStatusText(if (entity.state == "off") context.getString(R.string.state_off) else context.getString(
                 R.string.state_on))
             control.setControlTemplate(
-                ToggleRangeTemplate(
-                    entity.entityId,
-                    entity.state != "off",
-                    "",
-                    RangeTemplate(
-                        entity.entityId,
-                        0f,
-                        100f,
-                        (entity.attributes["brightness"] as? Number)
-                            ?.toFloat()
-                            ?.div(255f)
-                            ?.times(100) ?: 0f,
-                        1f,
-                        "%.0f%%"
-                    )
-                )
+                    if ((entity.attributes["supported_features"] as Int) and 1 == 1)
```

Comment:
This some debugging logic I'm guessing?

Reply:
That's a bitwise operation. (Supported features AND 1) should be equal to one, it just looks a bit strange

Reply:
Can we add a constant so we know 1 the first bit is the brightness support?

Reply:
yes good idea, I added it now